### PR TITLE
Add small mcstas image files for testing.

### PR DESCRIPTION
**Pull request by YooSunYoung**
Related to #32 

Added two files for `open-beam`(background) and `sample`.

They are subset of actual files from [nextcloud](https://project.esss.dk/nextcloud/index.php/apps/files/?dir=/McStas_ODIN&fileid=18487919)

### CODE REVIEW

**Review by MridulS — APPROVED** [review]

File: src/ess/imaging/data.py [hunk 1]
```diff
@@ -4,19 +4,21 @@
...
         retry_if_failed=3,
-        base_url='https://public.esss.dk/groups/scipp/ess/imaging/',
+        base_url=f'https://public.esss.dk/groups/scipp/ess/imaging/{_version}/',
         version=_version,
         registry={
+            'small_mcstas_ob_images.h5': 'md5:2f181bbacb164c28bfaf7cce09701d92',
+            'small_mcstas_sample_images.h5': 'md5:3c42570951cabec7caedc76d90d03fa3',
             'small_ymir_images.hdf': 'md5:cf83695d5da29e686c10a31b402b8bdb',
-            'README.md': 'md5:0f375972d4008de6060b065ac13ba17f',
+            'README.md': 'md5:9e1beeb325f127d691a8d7882db3255d',
```
Comment (MridulS) [inline src/ess/imaging/data.py:21]: looks like there is no README anymore? https://public.esss.dk/groups/scipp/ess/imaging/1/

The tests are failing.

Comment (YooSunYoung) [reply src/ess/imaging/data.py:21]: @MridulS 
image

It is there. It just doesn't show up in the web-thingy. 

I'll see why it's failing though. It worked last time I made this change.

Comment (YooSunYoung) [reply src/ess/imaging/data.py:21]: Which tests are failing btw?

Comment (MridulS) [reply src/ess/imaging/data.py:21]: https://github.com/scipp/essimaging/actions/runs/11770808535/job/32783786918

Comment (MridulS) [reply src/ess/imaging/data.py:21]: Ahhh, the failure should go away after a release. The failure is at latest release which doesn't take into account the new url (with _version) and trying to find README files at the old location. Ignore me :)

**Pull request merged**